### DISCUSSION TITLE
How to switch different `to_json` and `from_json` between multiple different modes

### DISCUSSION THREAD

**Discussion by weypro** [nlohmann/json#4074] [Q&A]
For the same structure, I want to implement two different funtion groups to meet different needs. For example, I want to dump the json schema in a certain format and cotinue to use the plain json that can be  serialized or deserialized directly at the same time.
The format may be like `https://json-editor.github.io/json-editor/`

**Comment by nlohmann** [comment]
This is not possible with the current implementation.

**Reply by jzakrzewski** [reply]
:(
